app/controllers/director/kids_controller.rb

```diff
@@ -10,26 +10,22 @@ def new
 
   def create
     @kid = Kid.new(kid_params)
-
-    @parent = Parent.where("
-      user_id IN (
-        SELECT id FROM users
-        WHERE email = ?
-      )",
-      params[:parent_email]
-    )
-    @kid.parents << @parent
+    current_date = Date.today
+    age = ((current_date - @kid.birthdate).to_i)/365
```

Comment:
As Refactor no problem! I can make it in `application controller` and that was I have access to it anytime!